### PR TITLE
Upgrade the gifski library

### LINKED ISSUE #51: Update gifski library
@kornelski Would you be able to update https://github.com/ImageOptim/gifski/tree/master/gifski.xcodeproj to latest Xcode and then update the gifski library in this repo? I noticed it has had a bunch of improvements since the last time we updated: https://github.com/ImageOptim/gifski/compare/94d6339d917b908a2eacbca6eb29599f71d8f96e...master

### PR DESCRIPTION
**Pull request by sindresorhus**
Fixes #51 
Fixes part of #78

This only upgrades and adapts the new API. It does not switch to handling our own file writing. That can come later.

### CODE REVIEW

File: Gifski/GifskiWrapper.swift [hunk 1]
```diff
@@ -58,10 +58,6 @@ final class GifskiWrapper {
 		self.pointer = pointer
 	}
 
-	deinit {
-		gifski_drop(pointer)
```
Comment (sindresorhus) [inline Gifski/GifskiWrapper.swift]: @kornelski This was removed, so I guess there's no cleanup needed anymore? Should it do `pointer = nil` here?

File: Gifski/Gifski.swift [hunk 2]
```diff
@@ -19,10 +19,10 @@ final class Gifski {
...
 			case let .addFrameFailed(error):
 				return "Failed to add frame, with underlying error: \(error.localizedDescription)"
-			case let .endAddingFramesFailed(error):
-				return "Failed to end adding frames, with underlying error: \(error.localizedDescription)"
-			case let .writeFailed(error):
-				return "Failed to write to output, with underlying error: \(error.localizedDescription)"
+			case let .startWritingFailed(error):
+				return "Failed to start writing, with underlying error: \(error.localizedDescription)"
+			case let .finishWritingFailed(error):
+				return "Failed to finish writing, with underlying error: \(error.localizedDescription)"
```
Comment (sindresorhus) [inline Gifski/Gifski.swift]: @kornelski Are these error names correct?

File: Gifski/GifskiWrapper.swift (hunk 1 above)
Comment (kornelski) [reply Gifski/GifskiWrapper.swift]: Fine to remove. The cleanup is done by `gifski_finish()`

Comment (sindresorhus) [reply Gifski/GifskiWrapper.swift]: But what if it encounters an error and then `gifski_finish()` is not called.

File: Gifski/Gifski.swift (hunk 2 above)
Comment (kornelski) [reply Gifski/Gifski.swift]: The user-facing message doesn't need to distinguish between start/finish explicitly. The `gifski_finish()` call will report any error during writing.

Comment (sindresorhus) [reply Gifski/Gifski.swift]: Like this https://github.com/sindresorhus/Gifski/pull/90/commits/ec5905990312aceb079fcf522d0e603d821cf8d6 ?

**Pull request merged**